server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -4,9 +4,20 @@
 version: "3.8"
 services:
   cms:
-    image: taccwma/core-cms:ff737a3
+    image: taccwma/core-cms:latest
     volumes:
-      - ../cms/secrets.py:/code/taccsite_cms/secrets.py
+      - type: bind
+        source: ../cms/secrets.py
+        target: /code/taccsite_cms/secrets.py
+        read_only: true
+      - type: bind
+        source: ../cms/settings_local.py
+        target: /code/taccsite_cms/settings_local.py
+        read_only: true
```

Comment:
What's the advantage of this verbose config vs a read-only volume mount?

Reply:
**Apologies.** You reviewed **after** I added a commit (59a72f8) to test this verbose syntax inspired by [@taoteg's comment](https://github.com/TACC/Core-Portal/pull/1034#discussion_r1876693860). He hopes `:ro` would be adequate for the same effect, but [my testing (59a72f8) suggests is not adequate](https://github.com/TACC/Core-Portal/pull/1034#discussion_r1876731555).

**TL;DR:** This would, **if** user forgets settings or secrets file (e.g. if `curl`/`cp` fails), halt `docker compose … up` with an error. _I don't want to do this, because it is verbose._

Reply:
You are right, `ro` on its own does not seem to prevent empty folder creation locally